test/controllers/game_controller_test.exs

```diff
@@ -2,12 +2,8 @@ defmodule CoverMyPingPong.GameControllerTest do
   use CoverMyPingPong.ConnCase
 
   test "GET /matches", %{conn: conn} do
```

Comment:
I plan on writing tests later on. Wasn't sure if I should ditch them or just leave em since I'll be coming back at some point. Thoughts?